crates/consensus/consensus/src/lib.rs

```diff
@@ -396,7 +406,16 @@ impl ConsensusError {
     }
 }
 
+impl From<InvalidTransactionError> for ConsensusError {
+    fn from(value: InvalidTransactionError) -> Self {
+        Self::InvalidTransaction(value)
+    }
+}
+
+#[cfg(feature = "std")]
+impl std::error::Error for HeaderConsensusError {}
```

Comment:
please move it below the struct declaration

Reply:
done